daml-foundations/daml-ghc/tests/No_concat_clash.daml

```diff
@@ -0,0 +1,22 @@
+-- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its
+-- affiliates.  All rights reserved.
+
+-- This is a compile-time test.
+
+{-# LANGUAGE NoImplicitPrelude #-}
+
+-- Show that if the names of `Prelude` are not in scope, then there
+-- should be no ambiguity about `concat` introduced by `DA.Foldable`.
+
+daml 1.2
+module No_concat_clash where
```

Comment:
@shayne-fletcher-da Looks like somebody has an OCaml background. Or should I spell it O_caml? :stuck_out_tongue_winking_eye:

Reply:
Yes, I thought you might pick me up here for my "inflexibility" ( 😄  ). It's not my first violation in that folder!

Ooh - have you read [this](https://caml.inria.fr/pub/papers/xleroy-modular_modules-jfp.pdf)?